### ISSUE TITLE
Error: Cannot find module 'gulp-csscomb'

### ISSUE THREAD

**Issue by manasdk** [koistya/gulp-csscomb#18]
Pulling down v3.0.5 of gulp-csscomb led to the following failure -

Downgrade to v3.0.3 or v3.0.4 works.

```
$ npm install gulp-csscomb
npm WARN package.json fontello-update@0.6.1 No repository field.
npm http GET https://registry.npmjs.org/gulp-csscomb
npm http 304 https://registry.npmjs.org/gulp-csscomb
npm http GET https://registry.npmjs.org/csscomb
npm http GET https://registry.npmjs.org/through2
npm http 304 https://registry.npmjs.org/through2
npm http 304 https://registry.npmjs.org/csscomb
npm http GET https://registry.npmjs.org/readable-stream
npm http GET https://registry.npmjs.org/xtend
npm http GET https://registry.npmjs.org/commander/2.0.0
npm http GET https://registry.npmjs.org/csscomb-core/3.0.0-3.1
npm http GET https://registry.npmjs.org/gonzales-pe/3.0.0-28
npm http GET https://registry.npmjs.org/vow/0.4.4
npm http 304 https://registry.npmjs.org/xtend
npm http 304 https://registry.npmjs.org/commander/2.0.0
npm http 304 https://registry.npmjs.org/vow/0.4.4
npm http 304 https://registry.npmjs.org/gonzales-pe/3.0.0-28
npm http 304 https://registry.npmjs.org/readable-stream
npm http GET https://registry.npmjs.org/core-util-is
npm http GET https://registry.npmjs.org/inherits
npm http GET https://registry.npmjs.org/isarray/0.0.1
npm http GET https://registry.npmjs.org/process-nextick-args
npm http GET https://registry.npmjs.org/string_decoder
npm http 304 https://registry.npmjs.org/csscomb-core/3.0.0-3.1
npm http 304 https://registry.npmjs.org/string_decoder
npm http GET https://registry.npmjs.org/util-deprecate
npm http GET https://registry.npmjs.org/minimatch/0.2.12
npm http GET https://registry.npmjs.org/vow-fs/0.3.2
npm http 304 https://registry.npmjs.org/inherits
npm http 304 https://registry.npmjs.org/core-util-is
npm http 304 https://registry.npmjs.org/isarray/0.0.1
npm http 304 https://registry.npmjs.org/process-nextick-args
npm http 304 https://registry.npmjs.org/minimatch/0.2.12
npm http 304 https://registry.npmjs.org/vow-fs/0.3.2
npm http 304 https://registry.npmjs.org/util-deprecate
npm http GET https://registry.npmjs.org/lru-cache
npm http GET https://registry.npmjs.org/sigmund
npm http GET https://registry.npmjs.org/glob/3.2.8
npm http GET https://registry.npmjs.org/node-uuid/1.4.0
npm http GET https://registry.npmjs.org/vow-queue/0.3.1
npm http 304 https://registry.npmjs.org/lru-cache
npm http 304 https://registry.npmjs.org/sigmund
npm http 304 https://registry.npmjs.org/glob/3.2.8
npm http 304 https://registry.npmjs.org/node-uuid/1.4.0
npm http 304 https://registry.npmjs.org/vow-queue/0.3.1
gulp-csscomb@3.0.5 node_modules/gulp-csscomb
├── through2@2.0.0 (xtend@4.0.0, readable-stream@2.0.1)
└── csscomb@3.1.8 (commander@2.0.0, vow@0.4.4, gonzales-pe@3.0.0-28, csscomb-core@3.0.0-3.1)

$ node
> require('gulp-csscomb')
Error: Cannot find module 'gulp-csscomb'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:280:25)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at repl:1:2
    at REPLServer.self.eval (repl.js:110:21)
    at Interface.<anonymous> (repl.js:239:12)
    at Interface.EventEmitter.emit (events.js:95:17)
    at Interface._onLine (readline.js:202:10)
    at Interface._line (readline.js:531:8)
```

**Comment by jbaxleyiii** [comment]
I can second this issue, looks like the latest release is missing the index.js

**Comment by sotayamashita** [comment]
I have a some issue

**Comment by sotayamashita** [comment]
why is latest release missing the file ? when he published it, did he untracked the index.js ?

**Comment by koistya** [comment]
Perhaps some breaking changes in `npm` packager.. is it fixed in 3.0.6?

**Comment by sotayamashita** [comment]
@koistya You are talking that it is npm's problem ?

**Comment by koistya** [comment]
@sotayamashita yeah, `npm publish` didn't include `index.js` into the package for some reason.

**Comment by sotayamashita** [comment]
@koistya I got it. I think everyone who use this library will have a trouble so please write this situation in README.md, If it's alright with you.

**Comment by koistya** [comment]
@sotayamashita I have published 3.0.6 which has this issue fixed (index.js is now included into the npm package).
